packages/react/src/analytics/integrations/GTM/eventsMapper.ts

```diff
@@ -1,15 +1,99 @@
 import {
+  type AnalyticsProduct,
   type EventData,
+  type EventProperties,
   EventType,
   PageType,
   type TrackTypesValues,
+  utils,
 } from '@farfetch/blackout-analytics';
 import { get } from 'lodash-es';
 import { getEventProperties, getProductData } from './utils.js';
 import type { EventMappers } from './types/index.js';
 
 const noPropertiesMappedFn = () => ({});
 
+/**
+ * Returns the total event value for GA4 ecommerce events.
```

Comment:
Fix this description please. Note that GTM does have the concept of "ecommerce" events like GA4 so I would change this description to be a bit more specific.

Reply:
Fixed! 👍🏼